cablelabs/transparent-security
# Fix optional device daemons feature for mininet

#### What does this PR do?
Fixes #66 
Ensures that the run_daemons flag which is responsible for creating heartbeat daemons on the hosts that send out 1 UDP packet every 3 seconds.
#### Do you have any concerns with this PR?
no
#### How can the reviewer verify this PR?
run mininet integration and ensure that the heartbeat daemons are started and sending packets
#### Any background context you want to provide?
Was doing some P4 changes and was not able to observe any packets so debugging was very difficult.
#### Screenshots or logs (if appropriate)
#### Questions:
- Have you connected this PR to the issue it resolves? Fixes #66 
- Does the documentation need an update? no
- Does this add new dependencies? no
- Have you added unit or functional tests for this PR? n/a
